.cirrus.yml

```diff
@@ -278,6 +278,10 @@ macos_aarch64_stable_debug_task:
                  -p iceoryx2-bb-lock-free
                  -p iceoryx2-bb-log
                  -p iceoryx2-bb-memory
+                 -p iceoryx2-bb-posix
                  -p iceoryx2-bb-system-types
                  -p iceoryx2-bb-testing
+                 -p iceoryx2-bb-threadsafe
+                 -p iceoryx2-bb-trait-tests
+                 -p iceoryx2-cal
                  --lib --bins --tests --no-fail-fast
```

Comment:
If all tests are running, these lines can be removed and `IOX2_COMMON_BUILD_DEBUG` can be replaced by `IOX2_COMMON_BUILD_AND_TEST_DEBUG`